akka-stream/src/main/scala/akka/stream/scaladsl/MergeLatest.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.scaladsl
+
+import akka.stream.{ Attributes, Inlet, Outlet, UniformFanInShape }
+import akka.stream.stage.{ GraphStage, GraphStageLogic, InHandler, OutHandler }
+
+import scala.collection.{ immutable, mutable }
+import scala.reflect.ClassTag
+import scala.language.higherKinds
+
+object MergeLatest {
```

Comment:
missing docs?

Reply:
Users will see docs of the companion in IDEs so having docs copied here or only here makes sense

Reply:
ok